step.yml

```diff
@@ -1,13 +1,26 @@
 title: "Change Working Directory for subsequent Steps"
-summary: |-
-  Set bitrise source directory (`BITRISE_SOURCE_DIR`) to a specified path.
+summary: Changes Bitrise source directory (`BITRISE_SOURCE_DIR`) to a specified path.
```

Comment:
I'd suggest removing the `(`BITRISE_SOURCE_DIR`)` from the summary. I know it was there originally, but it's a detail you are not particularly interested in when you are searching for this step in the step browser in the workflow editor. Wdyt?

Reply:
It's okay to me!